### DISCUSSION TITLE
Huge logcat files

### DISCUSSION THREAD

**Discussion by lucasrodella** [MustardChef/WSABuilds#158] [Q&A]
My c: has been filling up repeatedly and I found that WSA has been saving ~300gb of logcat files, is there anyway to fix this?

I get windows message saying I need to clear space. I look around and see that C:\Users\user\AppData\Local\Packages\MicrosoftCorporationII.WindowsSubsystemForAndroid_8wekyb3d8bbwe\LocalState\diagnostics\logcat is 300+gb. I delete it and it fills up again the next day or two.

**Comment by MustardChef** [comment]
300GB?????

**Reply by lucasrodella** [reply]
yes, this is bizarre, if I try to open the log, windows crashes because the log file is so large

**Comment by lucasrodella** [comment]
i found someone with same problem -> https://www.reddit.com/r/WSA/comments/11diibm/147_gb_logcat_files/

**Reply by MustardChef** [reply]
Try the official builds. If its the same issue with the official builds, please file a feedback hub report.
